lib/ansible/modules/network/dellos9/dellos9_config.py

```diff
@@ -20,38 +20,37 @@
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-ANSIBLE_METADATA = {'status': ['preview'],
-                    'supported_by': 'core',
-                    'version': '1.0'}
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'core'}
```

Comment:
Yes, you are correct it adds support for Persistent Connection work. The initial understanding was, all the modules which doesn't support Persistent Connection work, will be backward compatible and work with Ansible 2.3. What we found out what its not backward compatible. Hence added the support. 

Regarding the community->core, I believe that was the agreement between Dell & Ansible. If that is not the case let me know.